### ISSUE TITLE
Can't delete log files

### ISSUE THREAD

**Issue by macr1408** [woocommerce/woocommerce#20716]
**Describe the bug**
Cannot delete logs from the status page (WooCommerce -> Status -> Logs). When I click on Delete Log the page just reloads and nothing happens, I don't even receive an error. This is happening with the new versions of WooCommerce, it didn't happen before.

I tried with Apache and Nginx, but it happens anyway (actually, got this problem in more than 10 servers). 

**To Reproduce**
1. Install Wordpress.
2. Install Woocommerce.
3. Log anything.
4. Try to delete it.

**Expected behavior**
Log files being deleted.

**Isolating the problem (mark completed items with an [x]):**
- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```

### WordPress Environment ###

WC Version: 3.4.3
Log Directory Writable: ✔
WP Version: 4.9.7
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: nginx/1.10.3
PHP Version: 7.2.6-1+0~20180611145758.22+stretch~1.gbpe20e8b
PHP Post Max Size: 8 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.52.1
OpenSSL/1.0.2l

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.1.26-MariaDB-0+deb9u1
Max Upload Size: 2 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ❌ Your server does not have the SoapClient class enabled - some gateway plugins which use SOAP may not work as expected.
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.4.3
WC Database Prefix: wp_
MaxMind GeoIP Database: ✔
Total Database Size: 2.56MB
Database Data Size: 1.72MB
Database Index Size: 0.84MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.05MB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB
wp_comments: Data: 0.02MB + Index: 0.08MB
wp_erp_products: Data: 0.02MB + Index: 0.00MB
wp_erp_products_meta: Data: 0.05MB + Index: 0.00MB
wp_erp_stores: Data: 0.02MB + Index: 0.00MB
wp_links: Data: 0.02MB + Index: 0.02MB
wp_options: Data: 1.03MB + Index: 0.02MB
wp_postmeta: Data: 0.05MB + Index: 0.03MB
wp_posts: Data: 0.05MB + Index: 0.06MB
wp_termmeta: Data: 0.02MB + Index: 0.03MB
wp_terms: Data: 0.02MB + Index: 0.03MB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_usermeta: Data: 0.02MB + Index: 0.03MB
wp_users: Data: 0.02MB + Index: 0.05MB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB

### Post Type Counts ###

attachment: 2
page: 2
post: 2
product: 41
shop_order: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ❌Error messages should not be shown to visitors.

### Active Plugins (3) ###

Ecomerciar ERP: by Ecomerciar – 1.0
WooCommerce: by Automattic – 3.4.3
WP Crontrol: by John Blackbourn & contributors – 1.6.2

### Settings ###

API Enabled: –
Force SSL: –
Currency: ARS ($)
Currency Position: left
Thousand Separator: 
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Shop base: ❌ Page not set
Cart: ❌ Page not set
Checkout: ❌ Page not set
My account: ❌ Page not set
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 2.3.2
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –
```
</details>

**Comment by Ivcho-Varna** [comment]
Same problem here

**Comment by kloon** [comment]
Confirmed bug, the get_log_file_name method is being passed the full name including the date which is causing the issue.

**Comment by kloon** [comment]
#20730

**Comment by makeonlineshop** [comment]
hello, same problem here, I need to delete the logs files directly by FTP at
/wp-content/uploads/wc-logs

Will it be fixed soon ?
Thank you so much.

**Comment by kloon** [comment]
@makeonlineshop has been fixed in #20730 and will go out with 3.4.4 when we release that.
